tests/test_server.py

```diff
@@ -302,6 +314,20 @@ def test_pipInstall_authedOk(protected_server, package, pipdir):
     assert pipdir.join(package.basename).check()
 
 
+def test_requires_python(open_server, package, source_zip_package):
+    url = _build_url(open_server.port) + '/simple/centodeps/'
+    resp = urlopen(url)
+    assert resp.getcode() == 200
+    page_content = resp.read().decode('utf-8')
+    print(page_content)
+    assert 'data-requires-python="&gt;=4.2"' in page_content
+    # Should we depend on something like beautifulsoup to parse the HTML or does
```

Comment:
@mplanchard what do you think of this?

Reply:
@mplanchard I'm back from holidays and will be able to update this when you'll answer :)